src/Uno.UI.RuntimeTests/IntegrationTests/common/TestServices.InputHelper.cs

```diff
@@ -9,6 +9,7 @@
 using Uno.UI.RuntimeTests.Tests.Uno_UI_Xaml_Core;
 using System.Threading;
 using Windows.UI.Xaml.Media;
+using Uno.UI.RuntimeTests.Helpers;
```

Comment:
I think this change is not needed

Reply:
Hmm.. What is the error?

Reply:
Ah, seems it is missing the using on master too https://github.com/unoplatform/uno/runs/15750427713 . So somehow we merged it without the using 😅